docs/api-guide.md

```diff
@@ -697,17 +689,28 @@ Put put =
         .build();
 ```
 
-#### `Delete` operation
+##### Blind `Put` operation
 
-`Delete` is an operation to delete a record specified by a primary key.
+If you're certain that a record you're trying to write doesn't exist, you can use a blind `Put` operation for better performance. A blind `Put` is faster than a regular `Put` as it doesn't read the record beforehand. However, if the record does exist, the operation will fail due to a conflict.
```

Comment:
IIUC, the "blind write" feature in this PR is basically implicitly filling read-set if needed so that users don't need to manually read the target record in advance. But `com.scalar.db.api.PutBuilder.Buildable#blind()` is to skip the implicit read for insert operations. So it sounds a bit confusing to me. 

`com.scalar.db.api.PutBuilder.Buildable#create()`, `insert()`, `skipImplicitRead()` or something might be better instead of `com.scalar.db.api.PutBuilder.Buildable#blind()`?

Reply:
Good point. In this context, a `blind write` means preparing a record without having an existing record (in the read set). So when you set the `blind` option for the Put operation, we skip the implicit read. What are your thoughts on this?

Reply:
Sounds good.

But, when I read the description of this PR, I thought "blind writes" meant implicit filling read-set. So, I still feel some inconsistency between the description and `com.scalar.db.api.PutBuilder.Buildable#blind()`.

> This PR adds support for blind writes in Consensus Commit. Specifically, if a record is in the write set but not in the read set, ScalarDB will read the record and add it to the read set before preparing the records internally.

Reply:
Thanks for the detailed explanation. I think we are on the same page.

I also agree to the ideas of `user-perspective blind writes` and `transaction-manager-perspective blind writes`.

> Maybe we don't need to modify the option name blind. And should I update the PR description instead?

Maybe it's related to how we want to appeal this feature to users. If we publicly call `user-perspective blind writes` "blind write", the `blind()` option might confuse users. Otherwise, the description and release note (`With this enhancement, users can execute blind writes without the need to read records prior to writing them.`) section should be updated to clearly describe "blind write" means `transaction-manager-perspective blind writes` (plus "We don't need in-advance reads anymore when you write a fixed value!"?).

I want to hear opinion from @feeblefakie as a product owner as well.

Reply:
I've updated the description (and the release notes) slightly. Maybe we should also update the PR title but I didn't for now. Can you please check the description and let me know if anything still seems confusing?

Reply:
It looks good to me!